supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from conftest import Fields, TestFields
+from django.core.exceptions import FieldDoesNotExist, ValidationError, ObjectDoesNotExist
+from supplier.models import Supplier
+
+
+class TestSupplierModel:
+    @pytest.mark.django_db()
+    def test_save_supplier(self, supplier0):
+        assert supplier0 not in Supplier.objects.all()
+        Supplier.save_supplier(supplier0)
+        assert supplier0 in Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_supplier_error(self, supplier0):
+        with pytest.raises(ObjectDoesNotExist):
+            assert supplier0 not in Supplier.objects.all()
+            Supplier.delete_supplier(supplier0)
+
+    @pytest.mark.django_db()
+    def test_delete_supplier(self, supplier0):
```

Comment:
Maybe change the name of this function to `test_save_delete_supplier`? Or another, more informative, name.

Reply:
It is right he is testing delete functionallity so we dont need save since he tested that in line 9
test_delete_existed_supplier will be better